src/js/hora.js

```diff
@@ -335,196 +383,11 @@ define([
             }
         };
 
-        Hora.navigation = {
```

Comment:
@ericmuyser Can we have 2 copies of it? Like Hora 1.x points to old wiki and Hora 2.x points to new wiki.